Animations/CommonAnimationProps.js

```diff
@@ -0,0 +1,12 @@
+var CommonAnimationProps = {
+  enter: {
```

Comment:
can we have `CommonAnimationProps` for `enterActive` and `leaveActive`?  This is an honest question I'm not sure so if we can't that's totally fine I just wanted to present the idea.  Additionally, can we include the `opacity` in these props it seems like all the animations have the same `opacity` changes.